app/src/main/java/com/example/plantsapp/presentation/ui/MainActivity.kt

```diff
@@ -27,6 +35,11 @@ class MainActivity : AppCompatActivity() {
             }
             true
         }
+
+        startNotificationWorker(
```

Comment:
I think you can move this Work Exqueueing into Apllication

It has its own onCreate method